simulation/traffic_simulator/include/traffic_simulator/hdmap_utils/hdmap_utils.hpp

```diff
@@ -299,10 +318,16 @@ class HdMapUtils
 
   auto isTrafficLightRegulatoryElement(const lanelet::Id) const -> bool;
 
+private:
+  constexpr static double DEFAULT_MATCH_TO_LANE_REDUCTION_RATIO = 0.8;
```

Comment:
Please leave comment about what is the hard-coded value means, and why the value is `0.8`.

Reply:
I added comment in [`45d1c64` (#1458)](https://github.com/tier4/scenario_simulator_v2/pull/1458/commits/45d1c6410cccb755452b15983aab8992a99a1fbd)